test/Abp.ZeroCore.Tests/Zero/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -583,11 +579,8 @@ public void Should_Not_Write_History_For_Full_Audited_Entity()
         [Fact]
         public void Should_Not_Write_History_For_Not_Audited_Entities_Shadow_Property()
         {
-            Resolve<IEntityHistoryConfiguration>().IsEnabled = true;
-            Resolve<IEntityHistoryConfiguration>().Selectors.Clear();
```

Comment:
I cleared selectors to be sure there is no item in IEntityHistoryConfiguration,to be sure any Save method trigger is caused because of shadow property. Why did you remove it?

Reply:
Also we have that line in other `Should_Not_Write_History_For_Not_Audited_Entities_Shadow_Property` test. 
https://github.com/aspnetboilerplate/aspnetboilerplate/pull/5151/files#diff-55ecd88ee207dc8e04b0302823bf206bR872

Reply:
https://github.com/aspnetboilerplate/aspnetboilerplate/blob/5251b8fc8d461e55b7bbe5a66d0c7a92333601c5/src/Abp/EntityHistory/EntityHistoryConfiguration.cs#L18-L19

EntityHistoryConfiguration.IsEnabled is `false` by default
EntityHistoryConfiguration.Selectors is empty by default

we dont need to clean up the configuration. If we want to be explicit about it, it should be done at the constructor for all the test cases.

https://github.com/aspnetboilerplate/aspnetboilerplate/blob/5251b8fc8d461e55b7bbe5a66d0c7a92333601c5/test/Abp.Zero.SampleApp.Tests/EntityHistory/SimpleEntityHistory_Test.cs#L49

Reply:
We can also remove https://github.com/aspnetboilerplate/aspnetboilerplate/pull/5151/files#diff-55ecd88ee207dc8e04b0302823bf206bR872. 👍

Reply:
Removal PR is at https://github.com/aspnetboilerplate/aspnetboilerplate/pull/5271